config/manifest.yaml

```diff
@@ -22,6 +22,7 @@ exporters:
 
 processors:
     - gomod: go.opentelemetry.io/collector/processor/batchprocessor v0.81.0
+    - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor v0.81.0
```

Comment:
Is this the only processor we want to do add or do we want to add the regular resource processor as well?

Reply:
ok, yea let's get another pr open after this one to get that processor in here https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourceprocessor - happy to pair on it